`'Library/KeypadDeviceImplLib.h' file not found`

Hello,

During the compilation the LLVM is trying to find `edk2-mtk/Silicon/MediaTek/MT6765Pkg/Library/KeypadDeviceImplLib.h` which is at `edk2-mtk/Silicon/MediaTek/MediaTekPkg/Include/Library/KeypadDeviceImplLib.h`.

Full error:
```cBuilding ... /source/edk2-mtk/Common/edk2/ArmPkg/Library/ArmGenericTimerPhyCounterLib/ArmGenericTimerPhyCounterLib.inf [AARCH64]
/source/edk2-mtk/Silicon/MediaTek/MT6765Pkg/Library/KeypadDeviceImplLib/KeypadDeviceImplLib.c +4:10: fatal error: 'Library/KeypadDeviceImplLib.h' file not found
#include <Library/KeypadDeviceImplLib.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [GNUmakefile:290: /source/edk2-mtk/workspace/Build/roo/RELEASE_CLANG38/AARCH64/Silicon/MediaTek/MT6765Pkg/Library/KeypadDeviceImplLib/KeypadDeviceImplLib/OUTPUT/KeypadDeviceImplLib.obj] Error 1


build.py...
 : error 7000: Failed to execute command
	make -s tbuild [/source/edk2-mtk/workspace/Build/roo/RELEASE_CLANG38/AARCH64/Silicon/MediaTek/MT6765Pkg/Library/KeypadDeviceImplLib/KeypadDeviceImplLib]
ls Library/KeypadDeviceImplLib.h
^C

build.py...
 : error 7000: Failed to execute command
	make -s tbuild [/source/edk2-mtk/workspace/Build/roo/RELEASE_CLANG38/AARCH64/libs/FreeType]

- Aborted -
Build end time: 13:08:56, Nov.28 2024
Build total time: 00:00:33

+ return 1
```

Full build log: [edk2-mtk.log](https://github.com/user-attachments/files/17948703/edk2-mtk.log)

--- a/Silicon/MediaTek/MT6765Pkg/Include/Library/KeypadDeviceImplLib.h
+++ b/Silicon/MediaTek/MT6765Pkg/Include/Library/KeypadDeviceImplLib.h
@@ -1,0 +1,35 @@
+#ifndef __LIBRARY_KEYPAD_DEVICE_IMPL_H__
+#define __LIBRARY_KEYPAD_DEVICE_IMPL_H__
+
+#include <Protocol/KeypadDevice.h>
+
+EFI_STATUS EFIAPI KeypadDeviceImplReset(KEYPAD_DEVICE_PROTOCOL *This);
+EFI_STATUS        KeypadDeviceImplGetKeys(
+           KEYPAD_DEVICE_PROTOCOL *This, KEYPAD_RETURN_API *KeypadReturnApi,
+           UINT64 Delta);
+
+
+typedef enum {
+  KEYSTATE_RELEASED,
+  KEYSTATE_PRESSED,
+  KEYSTATE_LONGPRESS_RELEASE,
+} KEY_STATE;
+
+typedef struct {
+  // keydata to be send to the driver
+  EFI_KEY_DATA KeyData;
+
+  // the time the current action has been running
+  UINT64 Time;
+
+  // the current state of the key
+  KEY_STATE State;
+
+  // the current key has to repeat sending events
+  BOOLEAN Repeat;
+
+  // the user did a longpress
+  BOOLEAN Longpress;
+} KEY_CONTEXT;
+
+#endif

